lib/primo/search.rb

```diff
@@ -51,7 +51,18 @@ def self.get(params = {})
       params = params.to_h.transform_keys(&:to_sym)
       method = get_method params
       (url, query) = url(params)
-      new super(url, query:, timeout: Primo.configuration.timeout(params)), method
+
+      retry_count = Primo.configuration.enable_retries ? Primo.configuration.retries : 0
+      begin
+        new super(url, query:, timeout: Primo.configuration.timeout(params)), method
+      rescue Net::ReadTimeout
```

Comment:
We should rescue anything not just Net::ReadTimeouts.  If retrying doesn't work it will fail the same way each time.